### PR TITLE
Code Review

### PR DESCRIPTION
**Pull request by ndjerrou**
Hi,

Would you kindly please check my project for the Apple's one?

Thanks a lot !

### CODE REVIEW

File: index.html
Comment (Oghenebrume50) [inline index.html]: This input should be declared properly with type and other attributes

File: style.css
Comment (Oghenebrume50) [inline style.css]: Fun fact @import is general slower than doing the regular <link href> in the html file, should you want to change that, that's a good motivation :smiley: but it is not required

File: style.css
Comment (Oghenebrume50) [inline style.css]: Please delete commented codes

**Review by Oghenebrume50 — CHANGES_REQUESTED** [review]
Very good start @ndjerrou kudos :clap: stickler is not yet active on this repository, please get that to work, make these few changes and submit for a re-review

File: index.html
Comment (ndjerrou) [reply index.html]: okay fulfilled the input

File: style.css
Comment (ndjerrou) [reply style.css]: Erased it , okay :)

File: .stickler.yml
Comment (bolah2009) [inline .stickler.yml]: #### `REQUIRED`

Add `.stickler.yml` file to the root of the project. Also let's use [this file](https://github.com/microverseinc/linters-config/blob/master/css/.stickler.yml). :point_down: 

``` yml
linters:
  stylelint:
    config: 'stylelint.config.js'
```
- Also, add the `stylelint.config.js` that was referenced above, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/css/stylelint.config.js)

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/css).

File: README.md
Comment (bolah2009) [inline README.md]: #### `HINT`

Add a descriptive readme. Details like assignment link (from Odin), live link (from [githack](https://raw.githack.com/)), screenshot and what was learned during the project can make the project more descriptive. :+1:

**Review by bolah2009 — CHANGES_REQUESTED** [review]
# Awesome Job @ndjerrou 

## The [clone](https://raw.githack.com/ndjerrou/Apple_Clone/f2b93dbdbef34b5f84e1c49c88a409f2bc4f03eb/index.html) looks nice and uses CSS Backgrounds effectively. :ok_hand: 


### However, the navbar content should all have the gradient effect. see below:
| Clone | Source |
| :-----: | :-----: |
| image | image |

Before giving a :+1: let's make some **required** changes. :blush: 
- Add appropriate linter file. [See the comment on linter below]()
   > [Set up a code linter](https://github.com/microverseinc/linters-config/tree/master/css).
- [Next Project] Use GitHub Flow: For HTML & CSS projects, use only one feature branch (e.g. `embedding-images-and-video`, `html-forms`, `responsive-design`, etc) and **do not merge any feature branch to master until it's been approved**.
   > Follow the [Github Flow guidelines](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/).
- :bug: Fix navbar
- :book: Add a descriptive readme (See suggestions in comment below) 
   > Remember about descriptive README file.
- :wrench:  Resolve merge conflicts 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: .stickler.yml
Comment (ndjerrou) [reply .stickler.yml]: > Add .stickler.yml file to the root of the project

Do u mean that I have to moove those file to the master Branch ?

Because I've for the moment all of them in the 'dev' branch

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Review by Torres-ssf — APPROVED** [review]
### Hi there @ndjerrou :hand: 
Nice work on this page, it looks awesome :clap: :tada: 
I'm giving this project a :+1: 
Nice work implementing the suggested changes.
Good luck with the next one. :smile: 
Happy coding :computer: 

[FOR FUTURE PROJECT]
  As it said in the pathwrite, you should use Github Flow in your projects. So give a feature name to your branch instead of developing/developer/dev. You can read more about this [here](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/).

**Pull request merged**
